### PR TITLE
Relogin functionality added

### PR DESCRIPTION
**Pull request by sanbornsen**
Any currently any 401 response redirects the user to the login page. 

We still need to find a better practice on UX and UI code as well.

### CODE REVIEW

**Comment by sanbornsen** [comment]
[test]

**Comment by alexeykazakov** [comment]
I wonder if calling /api/login/authorize directly instead of routing to the login page would be better approach. No need to click on login button for the user and even automated token re-issuing may work.

File: src/app/auth/authentication.service.ts [hunk 1]
```diff
@@ -31,11 +31,13 @@ export class AuthenticationService {
     return false;
   }
 
-  logout() {
+  logout(redirect: Boolean = false) {
```
Comment (joshuawilson) [inline src/app/auth/authentication.service.ts]: this should be lowercase 'boolean' to be a type

**Comment by sanbornsen** [comment]
@alexeykazakov I thought the same, it's just am an approach to fixing the breakdown in production. Once we have the accepted behavior we can implement it the right way. What do you say?

**Comment by sanbornsen** [comment]
https://github.com/fabric8io/fabric8-planner/issues/941

**Comment by alexeykazakov** [comment]
@sanbornsen, agreed.

**Pull request merged**